bucket/openhv.json

```
@@ -0,0 +1,36 @@
+{
+    "version": "20210321",
+    "description": "Open Hard Vacuum. 90s Pixelart Science-Fiction Real-time strategy game with multiplayer support, competent skirmish AI and an integrated map editor",
+    "homepage": "https://openhv.itch.io/openhv",
+    "license": "GPL-3.0,...",
```

OpenRA is under GPLv3+. As for asset licenses, I generally don't bother listing them, so this should be OK:

```suggestion
    "license": "GPL-3.0-or-later",
```